sdk/mongodbatlas/Azure.ResourceManager.MongoDBAtlas/CHANGELOG.md

```diff
@@ -1,14 +1,23 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0
 
-### Features Added
+### Features Added (2025-06-30)
```

Comment:
the release date should follow the version number

Reply:
in addition, you need to update the version in csproj as well.
and in the README.md file there is a command of how to install this package, we need to remove the `--prerelease` option (try just search this in the file)